wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -25,37 +26,41 @@ import (
 	"github.com/offchainlabs/nitro/arbutil"
 )
 
+var (
+	HTTPHeaderCloudflareConnectingIP  = textproto.CanonicalMIMEHeaderKey("CF-Connecting-IP")
```

Comment:
This will lowercase the "F" in "CF", right? We should do a case insensitive comparison when using this variable, which it looks like we currently don't.

Reply:
The headers that the websocket library sees are always canonicalized by the go networking libraries, that's why I did it this way. But I could equally do this as a case insensitive comparison.

Reply:
I'll make it case insensitive so we aren't relying on an implementation detail.